cmd/acra-server/acra-server.go

```diff
@@ -265,16 +267,21 @@ func main() {
 
 		ocspClientVerifier := network.DefaultOCSPVerifier{Config: *ocspClientConfig, Client: &network.DefaultOCSPClient{}}
 
-		crlConfig, err := network.NewCRLConfig(*tlsCrlUrl, *tlsCrlFromCert)
+		var crlClientConfig *network.CRLConfig
+		if len(*tlsCrlClientUrl) > 0 {
```

Comment:
why not compare with empty string? `*tlsCrlClientUrl != ""`, because more often we check by len() values which are some containers and if not to look what type is of tlsCrlClientUrl variable, a reader may think that it's some array of strings.